src/main/resources/templates/footer.html

```diff
@@ -0,0 +1,13 @@
+<html  xmlns="">
```

Comment:
Are you sure this is right? please use F12 to look at the page in the browser - is there more than one html tag?